keyboards/smart/smart70/info.json

```diff
@@ -0,0 +1,127 @@
+{
+    "keyboard_name": "Smart70Pro",
+    "usb": {
+        "pid": "0x0070",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+
+    "matrix_pins": {
+        "cols": ["A9", "A8", "B15", "B14", "B13", "B12", "B11", "B10", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3"],
+        "rows": ["A15", "B3", "B4", "B5", "A2"]
+    },
+
+    "indicators": {
+        "caps_lock": "B6",
+        "on_state": 0
+    },
+
+    "rgblight": {
+        "led_count": 4,
+        "pin": "A10",
```

Comment:
This will need to be changed to `ws2812.pin` on develop.

Reply:
This is what I get now :
rgblight: Additional properties are not allowed ('ws2812.pin' was unexpected)

Reply:
`rgblight.pin` has been changed to `ws2812.pin` on develop; your PR is going to master, so you can't change it here.

Reply:
understand, so what shall I do now?  I goto to master because of VIA